docs/_source/practical_guides/create_dataset.md

```diff
@@ -256,29 +286,47 @@ ds.questions.pop(0)
 ```
 :::
 
-:::{tab-item} Guidelines
+:::{tab-item} Metadata
 This works for both local and remote `FeedbackDataset` instances.
 ```python
-# Define new guidelines from the template
+# Add metadata to the dataset
 ds = rg.FeedbackDataset(...)
 
-# Define new guidelines for a question
-ds.questions[0].description = 'New description for the question.'
+metadata = rg.TermsMetadataProperty(name="metadata", values=["like", "dislike"])
+
+ds.add_metadata_property(metadata)
+
+# Delete a metadata property
+ds.delete_metadata_properties(metadata_properties="metadata")
 ```
 :::
 
-:::{tab-item} Metadata
+:::{tab-item} Vectors
 This works for both local and remote `FeedbackDataset` instances.
 ```python
-# Add metadata to the dataset
 ds = rg.FeedbackDataset(...)
 
-metadata = rg.TermsMetadataProperty(name="metadata", values=["like", "dislike"])
+# Add vector settings to the dataset
+ds.add_vector_settings(rg.VectorSettings(name="my_new_vectors", dimensions=786))
 
-ds.add_metadata_property(metadata)
+# Change vector settings title
+vector_cfg = ds.vector_settings_by_name("my_vector")
+vector_cfg.title = "Old vectors"
+ds.update_vector_settings(vector_cfg)
```

Comment:
```suggestion
ds.update_vectors_settings(vector_cfg)
```

Reply:
What about `vector_cfg = ds.vector_settings_by_name("my_vector")`? Is this correct or also with an s? 
The fact that some methods/arguments are `vectors_settings` and others are `vector_settings` is quite confusing and prone to error. I understand that some take a list while others take a single `VectorSettings` object, but still we should have probably thought about a better naming for this.

Reply:
Yes , that's correct. I think the name of `vector_settings` combining singular and plural can be confusing. I agree with you. If you consider we can use `ds.vector_settings` for working with a list or a single vector settings, we can change this.